.devcontainer/devcontainer.json

```diff
@@ -1,14 +1,15 @@
 {
   "name": "Azure CLI Dev Container",
   "build": {
-    "dockerfile": "Dockerfile"
+    "dockerfile": "Dockerfile",
+    "context": ".."
   },
   "features": {
     "ghcr.io/devcontainers/features/github-cli:1": {}
   },
-  "workspaceFolder": "/workspaces",
-  "onCreateCommand": "uv venv .venv --seed",
-  "postCreateCommand": "REPO_NAME=$(basename $GITHUB_REPOSITORY) && cat $REPO_NAME/.devcontainer/init.sh >> ~/.bashrc && mkdir .vscode && cp $REPO_NAME/.devcontainer/mcp.json .vscode/",
+  // "workspaceFolder": "/workspaces",
+  "onCreateCommand": "uv venv .venv --seed --clear",
+  "postCreateCommand": "cat .devcontainer/init.sh >> ~/.bashrc && mkdir -p .vscode && cp .devcontainer/mcp.json .vscode/",
```

Comment:
CC @necusjz to review this change

Reply:
`"workspaceFolder": "/workspaces"` is by design. although the entry point of codespaces is `azure-cli`, in practice the abstraction layer sits above that. It is not just about operating on `azure-cli` repo, but also involves `azure-cli-extensions`, `azure-rest-api-specs`, and `aaz`.

in addition, changes to `.devcontainer` seem out of the scope of this pull request.